app/controllers/placements/schools/placements/add_placement_controller.rb

```diff
@@ -23,12 +23,16 @@ def update
       placement = @wizard.create_placement
       Placements::PlacementSlackNotifier.placement_created_notification(@school, placement.decorate).deliver_later
       @wizard.reset_state
-      redirect_to placements_school_placements_path(@school), flash: { success: t(".success") }
+      redirect_to update_path(@school), flash: { success: t(".success") }
     end
   end
 
   private
 
+  def update_path(school)
+    placements_school_placements_path(school)
+  end
```

Comment:
Minor nitpick about the naming of this method. I'm not sure `update_path` makes sense in this context, because it's actually pointing to the placements index route rather than an 'update' route.

Would it be more intention revealing if it was called something like `after_create_path`?

That would follow the example of existing methods like `after_sign_in_path` and `after_sign_out_path`:

https://github.com/DFE-Digital/itt-mentor-services/blob/de9aa3de7e51481425368ccdcf999ba82126af51/app/controllers/application_controller.rb#L30-L38

https://github.com/DFE-Digital/itt-mentor-services/blob/de9aa3de7e51481425368ccdcf999ba82126af51/app/controllers/application_controller.rb#L46-L48